daycare/forms.py

```diff
@@ -0,0 +1,31 @@
+from django import forms
+from .models import DayCare, Area
+from django.db.models import Max, Min
+
+
+class DayCareSearchForm(forms.Form):
+    min_price = DayCare.objects.all().aggregate(Min('price_per_day')).get('price_per_day__min')
+    max_price = DayCare.objects.all().aggregate(Max('price_per_day')).get('price_per_day__max')
+    start_date = forms.DateField(required=True, label='  Start date',
+                                 widget=forms.widgets.DateInput(attrs={'class': 'form-control', 'type': 'date'}))
+    end_date = forms.DateField(required=True, label='  End date',
+                               widget=forms.widgets.DateInput(attrs={'class': 'form-control', 'type': 'date'}))
+
+    area = forms.ChoiceField(required=False, label='Area',
+                             choices=(Area.choices + [('', 'All'), ]), initial="")
+    city = forms.CharField(required=False, label='City')
```

Comment:
Don't we have a list of cities? a free-text would be tough.

Reply:
Please see https://github.com/redhat-beyond/Hotails/pull/89#issuecomment-1120235525 and let me know what you think.

Reply:
Can you sum it up? I'm not sure I'm following. Anyway free-text can lead to issues (e.g: פתח-תקוה, פתח תקווה, פתח-תקווה, פתח תקוה)

Reply:
List of cities seems to me good solution for static-users ,but because all users are static we can control the cities input and just avoid mistakes like that.
If we will want to support registration in the future it might limit us . 
I saw django-cities(see https://pypi.org/project/django-cities/), it seems to have benefits of global support or even local but with more percise data.
I have no problem to create list of cities just want to know your opinion about it . @Omeramsc

Reply:
The UI is an entry point for this app and as such it should perform validation on the input,
regardless if the users are static or not.
Nothing prevents for the next dev to create another static user and enter non valid city name, right?
If this is a lot of work you can do it on a follow up PR and write in the PR description about it.